service/src/main/java/bio/terra/workspace/service/resource/WsmResourceService.java

```diff
@@ -42,9 +41,7 @@ public WsmResourceService(
     workspaceService.validateWorkspaceAndAction(
         userRequest, workspaceId, SamConstants.SAM_WORKSPACE_READ_ACTION);
 
-    return DbRetryUtils.throwIfInterrupted(
-        () ->
-            resourceDao.enumerateResources(
-                workspaceId, resourceType, stewardshipType, offset, limit));
+    return resourceDao.enumerateResources(
```

Comment:
We do not want flights to fail because we are restarting a pod. That is why I need to make a compensating fix in Stairway.
I just have to figure out how to include the exception check without hauling in piles of Spring stuff...